Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:13-buster
```

Comment:
@raz0red Curious your thoughts on this. I was doing some docker testing on the main webrcade, and could only get it to build dependencies properly on node-13, so I'm using it here as well. Does that track?

Reply:
I started on a script last night that will do the full build. I am actually using Docker to test it, so it will be pretty easy to package it as an image when it is complete.

Unfortunately, I need to resolve a couple of URL references. I think those are all cleaned up now, so I should have something posted late tonight or tomorrow morning for you to checkout.

Reply:
As far as the node versions go, here is what I have been using:
node: v12.13.0
npm: 6.12.0

Reply:
My though is to deliver the following:
1.) base build script that can be used on linux (assuming you have all the repos checked out correctly)

2.) Dockerfile that does both the build (phase 1) and delivers an image (phase 2) that includes Apache http. It also exposes a volume mount if you want to provide some content. Also, thinking of adding a simple PHP file that acts as a CORS proxy.

3.) Automated build in Git that just outputs the build package that you could extract and serve via Python, node, etc.

Reply:
Would love to help test! Let me know what I can do to help

Reply:
Great! thanks a lot. I will update this once I have something going and we can discuss how to test, etc.

Reply:
Here is the build script so far:
https://github.com/webrcade/webrcade/blob/build/dist.sh

I still have to make some changes to it, but it does build everything (assuming you have all the required repos cloned). 

Tomorrow I need to work on removing the direct references to the web site (so they refer to the local server). I am also going to drop the default feed and some other things from the local builds. 

Once I get those tweaks made, I will publish to the staging server and let you know.

Thanks!

Reply:
Hey-
I have the Docker image working. I need to tweak a couple of things, and then it is ready.

I created a new issue to track it: https://github.com/webrcade/webrcade/issues/13

If you are interested in helping me test the image, please make a comment on that issue and I will post instructions for pulling the image, etc.

Thanks!